pkg/version/VERSION

```diff
@@ -1 +1 @@
-default-version
+v0.1.0
```

Comment:
The modification of the version file cannot be submitted, and it will generate a real version number when release

Reply:
@elliotxx can we add this to .gitignore?

Reply:
@ruquanzhao We need to test it. Unlike regular .gitignore, we need VERSION in the remote repository (to ensure it can be pulled every time), while ignoring its local changes.

Reply:
i have rollback this fix. .gitignore if file exist it will not works